apps/omg_performance/lib/omg_performance/byzantine_events.ex

```diff
@@ -217,7 +217,6 @@ defmodule OMG.Performance.ByzantineEvents do
   defp map_contract_transaction(enumberable, transaction_function) do
     enumberable
     |> Enum.map(transaction_function)
-    # NOTE: infinity doesn't work, hence the large number
```

Comment:
... well... I too would like an infinite CPU machine but all I got is this lousy T-SHIRT?